lib/block-editor-settings.php

```diff
@@ -15,6 +15,65 @@
  * @return array New block editor settings.
  */
 function gutenberg_get_block_editor_settings( $settings ) {
+	// Recreate global styles.
+	$global_styles = array();
+	$presets       = array(
+		array(
+			'css'            => 'variables',
+			'__unstableType' => 'presets',
+			'isGlobalStyles' => true,
+		),
+		array(
+			'css'            => 'presets',
+			'__unstableType' => 'presets',
+			'isGlobalStyles' => true,
+		),
+	);
+	foreach ( $presets as $preset_style ) {
+		$actual_css = gutenberg_get_global_stylesheet( array( $preset_style['css'] ) );
+		if ( '' !== $actual_css ) {
+			$preset_style['css'] = $actual_css;
+			$global_styles[]     = $preset_style;
+		}
+	}
+
+	if ( wp_theme_has_theme_json() ) {
+		$block_classes = array(
+			'css'            => 'styles',
+			'__unstableType' => 'theme',
+			'isGlobalStyles' => true,
+		);
+		$actual_css    = gutenberg_get_global_stylesheet( array( $block_classes['css'] ) );
+		if ( '' !== $actual_css ) {
+			$block_classes['css'] = $actual_css;
+			$global_styles[]      = $block_classes;
+		}
+
+		/*
+		 * Add the custom CSS as a separate stylesheet so any invalid CSS
+		 * entered by users does not break other global styles.
+		 */
+		$global_styles[] = array(
+			'css'            => wp_get_global_styles_custom_css(),
```

Comment:
I renamed the method in the `compat/wp-6.2` directory with a guard clause, so we don't have to maintain a separate copy forever.

Reply:
:thinking: Isn't the point of this PR to make sure we use the gutenberg method that points to the gutenberg class?